src/main/java/net/atos/zac/configuratie/ConfiguratieService.java

```diff
@@ -107,6 +107,8 @@ public List<String> readAdditionalAllowedFileTypes() {
     @ConfigProperty(name = "CONTEXT_URL")
     private String contextUrl;
 
+    private String gemeenteCode = "0599"; // TODO configuration GEMEENTE_CODE instead of hard coded
```

Comment:
Moet hier nog een User Story voor komen?

Reply:
Komt in het volgende pull request.